src/clapsolver/dynamic_lsap.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+
+
+def augment(cost_matrix, curRow, row4col, col4row, u, v):
```

Comment:
change variable names to `snake_case` throughout or whatever that format is called.